### PR TITLE
[WIP] Add test for Aerodynamic Surface functions

### PR DESCRIPTION
**Pull request by CassianoMontibeller**

### CODE REVIEW

File: ADR/Components/Aerodynamic_components/tests/test_Aerodynamic_surface.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from ADR.Components.Aerodynamic_components.Aerodynamic_surface import Aerodynamic_surface
+import numpy as npt
+
+
+def test_Aerodinamics_surface():
+    airfoil_Parameters = {"airfoil_clmax": 1.2, "airfoil1_name": "s1223", "airfoil2_name": "s1223",
```
Comment (rafaellehmkuhl) [inline ADR/Components/Aerodynamic_components/tests/test_Aerodynamic_surface.py:6]: With a clmax of 1.2, the airfoil will not reach 20 degrees of angle of attack. See for yourself.
This is a sign (coming from the test, nice) that we should refactor the method itself, as it has problems.

File: ADR/Components/Aerodynamic_components/tests/test_Aerodynamic_surface.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                          "airfoil3_name": "s1223", "span1": 2, "span2": 1.5, "chord1": 0.40, "chord2": 0.30,
+                          "chord3": 0.35, "twist1": 0, "twist2": 0, "twist3": 0, "incidence": 0}
+    aerodynamics_surfase = Aerodynamic_surface(airfoil_Parameters)
+    npt.assert_almost_equal(
+        aerodynamics_surfase.update_alpha(20), 20, decimal=0)
```
Comment (rafaellehmkuhl) [inline ADR/Components/Aerodynamic_components/tests/test_Aerodynamic_surface.py:11]: This will throw an error as the update_alpha() method returns nothing.

With a clmax of 1.2 (from the parameters) the airfoil will not reach 20 degrees of angle of attack. See for yourself.
This is a sign (coming from the test, nice) that we should refactor the method itself, as it has problems.
This is also a sign that we should test this update_alpha function for more than one case (negative alpha, positive alpha, large positive and negative alpha, 0, etc).

File: ADR/Components/Aerodynamic_components/tests/test_Aerodynamic_surface.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+from ADR.Components.Aerodynamic_components.Aerodynamic_surface import Aerodynamic_surface
+import numpy.testing as npt
+import pytest
+
+
+@pytest.fixture
+def aerodynamic_surfase():
```
Comment (rafaellehmkuhl) [inline ADR/Components/Aerodynamic_components/tests/test_Aerodynamic_surface.py:7]: Typo: surfaCe

**Comment by rafaellehmkuhl** [comment]
The tests are good. Travis is failing because of a problem with the AVL Wrapper:

FileNotFoundError: [Errno 2] No such file or directory: '/home/travis/build/CeuAzul/ADR/ADR/Methods/VLM/AVL/outputs/cx_span.txt' -> '/home/travis/build/CeuAzul/ADR/ADR/Methods/VLM/AVL/outputs/cx_span.txt.bak'

We will hold this PR for now and accept it after we solve #142

**Comment by CassianoMontibeller** [comment]
Solves #156

**Comment by rafaellehmkuhl** [comment]
I see that the tests are still failing, but taking a look [on the log](https://travis-ci.com/github/CeuAzul/ADR/builds/174342710), its clear that this is because of the change on the AVL wrapper.

You inserted values that were calculated by the former wrapper, and as we saw on #175, the results have a slight change on the third decimal case, which has something to do with some wrapper configuration, but is so small that we decided not to deal with.

I ask that you update the assert values and run pytest locally so we can continue on this one.

File: ADR/Components/Aerodynamic_components/tests/test_Aerodynamic_surface.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+def test_update_alpha(aerodynamic_surface):
+    aerodynamic_surface.update_alpha(20)
+    new_alpha = aerodynamic_surface.attack_angle
+    npt.assert_almost_equal(new_alpha, 20)
+
+
+def test_get_alpha_range(aerodynamic_surface):
+    x = [-10, -9, -8, -7, -6, -5, -4, -3,
+         -2, -1, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
```
Comment (rafaellehmkuhl) [inline ADR/Components/Aerodynamic_components/tests/test_Aerodynamic_surface.py:23]: You can change it for:

```
x = list(range(-10, 11))
```

If you want to know more about the range iterator, take a look [here](https://stackoverflow.com/questions/18265935/python-create-list-with-numbers-between-2-values).
